com.unity.multiplayer.mlapi/Runtime/Messaging/CustomMessageManager.cs

```diff
@@ -35,7 +35,17 @@ internal CustomMessagingManager(NetworkManager networkManager)
         /// </summary>
         public event UnnamedMessageDelegate OnUnnamedMessage;
 
-        internal void InvokeUnnamedMessage(ulong clientId, Stream stream) => OnUnnamedMessage?.Invoke(clientId, stream);
+        internal void InvokeUnnamedMessage(ulong clientId, Stream stream)
+        {
+            var bytesCount = 0UL;
```

Comment:
Could potentially abstract this out of this method to reduce complexity here

Reply:
Possible tech debt for MLAPI: Should we be using raw .net stream classes instead of having proper abstractions over it? So that safeties like this can be centralized (or non-existent if we can make assumptions)